### PR TITLE
Add boilerplate xcvm module

### PR DESCRIPTION
**Pull request by natebeauregard**
- Adds a boilerplate no-op implementation for the `xcvm` module
- Created placeholder `.proto` files for `genesis`, `query` and `tx` and generated the related `*.pb.go` files

### CODE REVIEW

File: proto/composable/xcvm/v1beta1/query.proto [hunk 1]
```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+package composable.xcvm.v1beta1;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "x/xcvm/types";
+
+// Query provides defines the gRPC querier service.
```
Comment (natebeauregard) [inline proto/composable/xcvm/v1beta1/query.proto]: typo:
```suggestion
// Query provides the gRPC querier service.
```

File: x/xcvm/module.go [hunk 2]
```diff
@@ -0,0 +1,145 @@
+package tx_boundary
```
Comment (natebeauregard) [inline x/xcvm/module.go]:
```suggestion
package xcvm
```

File: x/xcvm/types/codec.go [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+	ModuleCdc = codec.NewAminoCodec(amino)
+)
+
+func init() {
+	RegisterLegacyAminoCodec(amino)
+	cryptocodec.RegisterCrypto(amino)
+
+	RegisterLegacyAminoCodec(authzcodec.Amino)
+	RegisterLegacyAminoCodec(govcodec.Amino)
+	amino.Seal()
```
Comment (joshklop) [inline x/xcvm/types/codec.go]: Why are these necessary?

File: x/xcvm/types/codec.go [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/codec/types"
+	cryptocodec "github.com/cosmos/cosmos-sdk/crypto/codec"
+	authzcodec "github.com/cosmos/cosmos-sdk/x/authz/codec"
+	govcodec "github.com/cosmos/cosmos-sdk/x/gov/codec"
+)
+
+var (
+	amino     = codec.NewLegacyAmino()
+	ModuleCdc = codec.NewAminoCodec(amino)
```
Comment (joshklop) [inline x/xcvm/types/codec.go]: Is this used anywhere?

File: x/xcvm/types/codec.go (hunk 3 above)
Comment (natebeauregard) [reply x/xcvm/types/codec.go]: Good point - seems to just be a noop. I was basing the majority of this approach off of the `x/tx-boundary` module and it looks like it's a [noop there as well](https://github.com/ComposableFi/composable-cosmos/blob/release/v7.0.x/x/tx-boundary/types/codec.go). I'll remove this init section here for the `xcvm` module and remove the unused variables mentioned below and then we can always readd any of this logic if we end up needing it for our implementation.

File: x/xcvm/types/codec.go (hunk 4 above)
Comment (natebeauregard) [reply x/xcvm/types/codec.go]: Removed!
